h5p-bildetema/package.json

```diff
@@ -49,16 +42,15 @@
     "@testing-library/react": "^14.0.0",
     "@types/jquery": "^3.5.16",
     "@types/node": "^20.2.5",
-    "@types/react": "^18.2.7",
-    "@types/react-dom": "^18.2.4",
-    "@vitejs/plugin-react": "^4.0.0",
     "esbuild": "^0.17.19",
+    "eslint-config-custom": "*",
```

Comment:
When referencing internal packages, we specify the version `*` because we always want to use the latest version, no matter what it is.